include/seqan3/alignment/pairwise/edit_distance_unbanded.hpp

```diff
@@ -97,12 +91,31 @@ class pairwise_alignment_edit_distance_unbanded
     using result_value_type = typename align_result_selector<database_type, query_type, align_config_type>::type;
 
     //!\brief When true the computation will use the ukkonen trick with the last active cell and bounds the error to config.max_errors.
-    static constexpr bool use_max_errors = detail::MaxErrors<align_config_t>;
+    static constexpr bool use_max_errors = align_config_type::template exists<align_cfg::max_error>();
     //!\brief Whether the alignment is a semi-global alignment or not.
     static constexpr bool is_semi_global = traits_t::is_semi_global_type::value;
     //!\brief Whether the alignment is a global alignment or not.
     static constexpr bool is_global = !is_semi_global;
 
+    //!\brief Whether the alignment configuration indicates to compute and/or store the score.
+    static constexpr bool compute_score = align_config_type::template exists<align_cfg::result<with_score_type>>() ||
```

Comment:
I think this is superfluous. The minimum requirement is to compute the score. So you can be sure that it is enabled.

Reply:
There might be configurations there I don't need that